.claude/scripts/pm/init.sh

```
@@ -0,0 +1,159 @@
+# Create directory structure
+echo ""
+echo "📁 Creating directory structure..."
+mkdir -p .claude/prds
+mkdir -p .claude/epics
+mkdir -p .claude/rules
+mkdir -p .claude/agents
+mkdir -p .claude/scripts/pm
+echo "  ✅ Directories created"
+
+# Copy scripts if in main repo
+if [ -d "scripts/pm" ] && [ ! "$(pwd)" = *"/.claude"* ]; then
```

Shellcheck (0.10.0)

[error] [ .. ] can't match globs. Use [[ .. ]] or case statement.

(SC2081)